setup-machine-arktos.sh

```diff
@@ -146,6 +142,26 @@ sudo pip3 install netaddr
 sudo pip3 install kopf
 sudo pip3 install PyYAML
 
+#####################
+
+echo Setup: Install Containerd
+
+cd $HOME
+wget https://github.com/containerd/containerd/releases/download/v1.4.2/containerd-1.4.2-linux-amd64.tar.gz
+cd /usr
+sudo tar xvf $HOME/containerd-1.4.2-linux-amd64.tar.gz
+
+cd $HOME/go/src/k8s.io/arktos
+wget -qO- https://github.com/futurewei-cloud/containerd/releases/download/tenant-cni-args/containerd.zip | zcat > /tmp/containerd
+sudo chmod +x /tmp/containerd
+sudo systemctl stop containerd
+sudo mv /usr/bin/containerd /usr/bin/containerd.bak
+sudo mv /tmp/containerd /usr/bin/
+sudo systemctl restart containerd
```

Comment:
It's better to separate containerd replacement from the setup-machine-arktos.sh. 
Here, it's restarting the containerd, but we do not need run containerd during machine setup

Reply:
@click2cloud-amit If these have been fixed, please respond to each issue with a brief description and resolve the comment so that it does not appear pending action.

Reply:
Yes @vinaykul. 
I have removed steps to replace the containerd from setup-machine-arktos.sh as per @clu2xlu suggestion and moved it to separate shell script.